### ISSUE TITLE
questions about the validation part

### ISSUE THREAD

**Issue by ghoshaw** [microsoft/singleshotpose#102]
Hi, I run the project on my own dataset and the training process looks great. But , when I test the model on a image in validation set, the result is bad and the result changes every time. Do you know anything like this? Thank you!

**Comment by btekin** [comment]
Hi @ghoshaw, could you provide more details about the problem? Answers to following questions could be helpful:

- What does your training data look like, how large is it? 
- Did you follow all the steps here (https://github.com/Microsoft/singleshotpose/blob/master/label_file_creation.md) to create labels for your own dataset? 
- Do you train for single object or multiple object pose estimation? 
- How is the convergence of the network and what are the final accuracy numbers?
